app/helpers/course/announcements_helper.rb

```diff
@@ -0,0 +1,9 @@
+module Course::AnnouncementsHelper
+  # Returns the formatted title of announcements component.
+  #
+  # @param [Course::AnnouncementSettings] settings The settings of announcements component.
+  # @return [String|nil] The formatted title of announcements component, or nil if the title is nil.
+  def announcements_title(settings)
```

Comment:
If the helper is mixed in then there's no need to pass the settings object, because `@announcement_settings` points to the same variable.